AnnoyancesFilter/sections/popups.txt

```diff
@@ -98,6 +98,7 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+aktu.ac.in#$#body { overflow: visible!important; }
```

Comment:
Scrolling will be broken with AG for iOS and AG Content Blocker, all 3 rules should be CSS-rules:
```
aktu.ac.in#$#body { overflow: visible!important; }
aktu.ac.in#$##onload { display: none!important; }
aktu.ac.in#$#.modal-backdrop { display: none!important; }
```

Reply:
Should I add the twitter widget to social filters?

Reply:
Yes, just add the domain to 2 rules under this comment:
`! ##.twitter-timeline + widgets.js`

Reply:
@gork7777 
Please add to this rule too: `##.twitter-timeline`